pkg/internal/util/clientwatcher.go

```diff
@@ -55,7 +55,7 @@ func WithoutClientWatcher() ClientWatcherOption {
 }
 
 const (
-	defaultTimeout = 30 * time.Second
+	defaultTimeout = 60 * time.Second
```

Comment:
please don't increase the defaultTimeout for all in this PR.

Reply:
Why? Many tests are flaky due to the short timeout, not only on prow and my local machine, I don't want to type `\retest` all the time.

Reply:
30s is quite a lot of time. Are they flacky on everything or just some specific operation? If so we should only increase that slow operation, not everything.